utils/docker/Dockerfile.code_review

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright 2019, Intel Corporation
+#
+#
+
+# Pull base image
+FROM fedora:latest
+MAINTAINER Brian J. Murrell <brian.murrell@intel.com>
+
+RUN dnf install -y bandit codespell diffutils git \
+                   perl \
+                   python3-pip python3-pygithub python3-requests \
+                   ShellCheck
```

Comment:
Is the list of needed tools really this much less than it was with EL7?

Reply:
No EPEL for fedora, and all the python modules are available as packages.
We don't have unit tests for the yaml/json/ruby linting.   That would require more packages and some rubygems.

Reply:
> all the python modules are available as packages

Is python3-pip still needed then?

Reply:
You were apparently looking at a stale source display.  The python3-pip was removed on the previous commit.